tests/kvasir-tests/Makefile.common

```diff
@@ -75,11 +78,24 @@ endif
 ifeq ($(OS_REL),26)
 	GOAL := goal-Fedora26
 endif
+# other versions of Fedora will use goal-Fedora if it exists
+# otherwise the default goal file - which may be incorrect?
 endif
 
+# Function to compute goal filename extension.
```

Comment:
Can you add at least a one-sentence comment saying what this is supposed to be doing?  That will help people reading it (or enable them to avoid reading the implementation).  Thanks.

Reply:
done